adapters/repos/db/aggregator/grouper.go

```diff
@@ -273,29 +272,6 @@ func (g *grouper) insertOrdered(elem group) {
 	}
 }
 
-// ScanAll iterates over every row in the object buckets
-// TODO: where should this live?
-func ScanAll(tx *bolt.Tx, scan docid.ObjectScanFn) error {
```

Comment:
The method was unused. As it still references bbolt, it smells like a very legacy code